app/models/invalid_identities.rb

```diff
@@ -0,0 +1,20 @@
+class InvalidIdentities
+
```

Comment:
@wtholt I have to agree, why a new model and not just a module or something in lib that inherits from identity model.  What is the benefit to this way? Since this is primarily used in a rake task you could also just extend the identity model in the rake task.